### DISCUSSION TITLE
Custom maps

### DISCUSSION THREAD

**Discussion by neyacinu** [smasherprog/EqTool#107] [Ideas]
Would love it if I could use my own custom map overlays like Nparse allows. In Nparse, if you record a map it goes into a folder:

nparse\data\maps\map_files\recordings\map_name.txt

It is extremely easy to use and overlay your own custom waypoints/etc on top of the real map!

**Comment by smasherprog** [comment]
This was just added. I had support for this the entire time, it was just never documented.
Custom Map loading is now supported. If you have a folder called     maps       this will be checked for map files first. If no map files are found, the program will fallback to the built in map files. You can add your overrides for zone files. If you are making changes, you need to clear the cache files. There is a button in the settings -> map tab to clear these. Or you can delete the cachemaps folder.

**Comment by smasherprog** [comment]
Discord is a better place to post these ideas.
https://discord.gg/Z6RPUMNcY2
